src/couch_replicator/src/couch_replicator_parse.erl

```diff
@@ -490,7 +490,7 @@ get_json_value(Key, Obj) ->
     couch_replicator_utils:get_json_value(Key, Obj).
 
 get_json_value(Key, Obj, Default) ->
-    couch_replicator_util:get_json_value(Key, Obj, Default).
+    couch_replicator_utils:get_json_value(Key, Obj, Default).
```

Comment:
Can we add a test for this as well?

Reply:
go right ahead.

Reply:
https://github.com/apache/couchdb/pull/4345